.npmignore

```diff
@@ -9,3 +9,10 @@ mocks.js
 
 # Development configs
 renovate.json
+
+# CI configs
+.github/
+
+# Docs
+images/
+**.md
```

Comment:
These are needed in the NPM website :(

Reply:
Nevermind, the README is pushed anyway and it's referencing the images via full GH URLs 👍